README.md

```diff
@@ -24,7 +24,9 @@ Workflows are continuously deployed to a [DNAnexus CI project](https://platform.
 
 ## Basic execution
 
-The easiest way to get started is on a single, Docker-capable machine (your laptop, shared workstation, or virtual machine) using [miniWDL](https://github.com/chanzuckerberg/miniwdl). MiniWDL can be installed either via `pip` or `conda` (via conda-forge). After confirming that it works (`miniwdl run_self_test`, you can use [miniwdl run](https://miniwdl.readthedocs.io/en/latest/getting_started.html) to invoke WDL workflows from this repository.
+<img src="https://github.com/openwdl/learn-wdl/blob/master/images/miniwdl-dev.png" width=600>
```

Comment:
Thank you for this PR; the image is a nice addition to the readme. Can it be sourced from the raw [user content domain](https://developer.github.com/changes/2014-04-25-user-content-security/) (i.e. `https://raw.githubusercontent.com/openwdl/learn-wdl/master/images/miniwdl-dev.png`) rather than `github.com`?

Reply:
updated to URL suggested